### DISCUSSION TITLE
404 error using three.js?

### DISCUSSION THREAD

**Discussion by TheR8R** [community/community#117138] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi. I am currently trying to host a small "game" if you will, it is more like a small simulation. It seems that Git struggles to find some of my javascript files. I saw post from other that had the same issues, but the solution to their problem didn't work here. 
![Screenshot 2024-04-06 120825](https://github.com/community/community/assets/90387698/b0046c2f-5188-4e78-afbb-40bd6a1838ec)
The repository is: https://ther8r.github.io/.

I recall seeing there being different limits based on your github subscription, but i can't find it again. It said something about a limit of 5, but i am not sure. Looking through the javascript files in the application tab, only 5 javascript files will load. not sure if it means anything
I understand the website has to be static, And it is. I use the Three.js library, but not sure if it has something to do with the way i import the library. When testing locally i used node.js and vite to run the website. Don't know if this proves to be an issue either.

**Comment by jfullstackdev** [comment]
both of it are 404, meaning it cannot access the files at the specified path, so you may want to check the correct URL

![image](https://github.com/community/community/assets/47092464/67e9ec8c-e0cb-452a-aac9-7c22528cca6a)

![image](https://github.com/community/community/assets/47092464/4e29e0b7-eb71-48e4-b70a-f4e58ffd4346)

![image](https://github.com/community/community/assets/47092464/642d634f-923d-4608-8f24-2d80ea91fab2)

**Reply by TheR8R** [reply]
I found an error in the way i declared the version. Thanks for the feedback. The issue of 404 not found, still persists, for 4 of my own files, specifically the keyboardControls.js, raycasters.js, soundImpl.js and pointerControls.js. I've no idea why git can't find them. I suspected it may have to do with the way i Import them, but i import other javascript classes and functions the same way, and they load just fine, like the player.js and levelLoader.js. 
![image](https://github.com/community/community/assets/90387698/8c1ba730-c085-4920-aa0a-df98ed19ff90)

![image](https://github.com/community/community/assets/90387698/e19a5e06-d7bc-488d-b881-39fd585b85ab)

**Reply by jfullstackdev** [reply]
feel free to ask more questions as far as I can help

**Reply by TheR8R** [reply]
What could be some of reasons for these 4 files not being found, while they are in the same folder as the levelLoader.js or player.js?
![image](https://github.com/community/community/assets/90387698/39654374-eebb-40d8-870c-c9d117e10e45)

**Reply by jfullstackdev** [reply, marked as answer]
there should be the .js, so you will be accessing it as that, like : 

https://ther8r.github.io/src/keyboardControls.js

because as you can see, they are all ending with that,

![image](https://github.com/community/community/assets/47092464/efe72d51-f29e-46e0-ba70-e288f28fdc08)

**Reply by TheR8R** [reply]
You my good sir, are fantastic at pointing out the obvious, which i am blind to. Thank you lots!
